ml_ops.sh

```diff
@@ -74,13 +74,13 @@ hdfs dfs -rm -R -f ${HDFS_SCORED_CONNECTS}
 
 # Add -p <command> to execute pre MPI command.
 # Pre MPI command can be configured in /etc/duxbay.conf
-# In this script, after the line after --mpicmd ${MPI_CMD} add:
-# --mpiprep ${MPI_PREP_CMD}
+# In this script, after the line after -c ${MPI_CMD} add:
```

Comment:
Sorry I overlooked the verbose name for that parameter. 
I couldn't get .jar to work with empty parameter but maybe I didn't try hard.

Will work in a solution.